Files/Filesystem/Drives.cs

```diff
@@ -58,9 +59,9 @@ private async void DeviceWatcher_EnumerationCompleted(DeviceWatcher sender, obje
 			{
 				await CoreApplication.MainView.Dispatcher.RunAsync(CoreDispatcherPriority.Low, () =>
 				{
-					if (App.sideBarItems.FirstOrDefault(x => x is HeaderTextItem && x.Text == "Drives") == null)
+					if (App.sideBarItems.FirstOrDefault(x => x is HeaderTextItem && x.Text == ResourceController.GetTranslation("Drives")) == null)
 					{
-						App.sideBarItems.Add(new HeaderTextItem() { Text = "Drives" });
+						App.sideBarItems.Add(new HeaderTextItem() { Text = ResourceController.GetTranslation("Drives") });
```

Comment:
Can you rename `Drives` to `SidebarDrives` in order to match the rest of the sidebar items?

Reply:
Yeah, i will rename it as soon as possible.